src/zkevm_specs/util/arithmetic.py

```diff
@@ -15,10 +15,13 @@ def linear_combine(seq: Sequence[Union[int, FQ]], base: FQ, range_check: bool =
     >>> assert linear_combine([1, 2, 3], r) == 1 + 2 * r + 3 * r**2
     """
     result = FQ.zero()
+    base_int = base.n if isinstance(base, FQ) else base
+    if base_int == 0:
+        return result
```

Comment:
This is a check that will never be performed in circuit. As is easier and cheaper to simply treat 0 as all other entries. 

Therefore, since the specs try to represent what we will run, I think this should not be included.

Reply:
This is added to fixup for many `RLC(0)` used in the spec.

Reply:
Maybe we can put a comment there to keep both ends work?

Reply:
Sure, that makes sense :smile: